src/sync/branch.test.ts

```diff
@@ -135,7 +135,6 @@ describe("sync/branches", () => {
 				.repoSyncStatePendingForBranches()
 				.create();
 
-			mocked(sqsQueues.backfill.sendMessage).mockResolvedValue(Promise.resolve());
```

Comment:
Test doesn't need anything from the mock result. So remove the mock impl instead of feeding real object, which need to comply to a certain ts structure.